challenges.tex

```diff
@@ -0,0 +1,540 @@
+insertion zone.
+\subsection{Mockup Machines}\label{sec:machines}
+In case no real \ac{MPS} stations are available, replications
+(so called \emph{mockup machines}) may be used, that do not need to
+physically perform the respective production steps. Instead, that work may
+be carried out by a human supervisor (see \refsec{sec:operators}).
+The minimum requirements for a mockup machine are specified in the following.
+
+Mockup machines are required to have the same box-like base-shape as specified
+in the RCLL rulebook.
+Additionally, the following parts need to be mounted:
+\begin{itemize}
+	\item a model of the conveyor belt
+	\item a shelf on the front right side of the box on stations replicating a \ac{CS}
+	\item either a shelf, a slide or a conveyor belt on the front right side
+of the box, such that it is accessible from the front on stations replicating a \ac{RS}
+\end{itemize}
+Models for a conveyor belt, shelf and slides can be found in the RCLL rulebook
+repository\footnote{\url{https://github.com/robocup-logistics/rcll-rulebook/tree/common/challenges/mock_up_models}}.
+
+The building materials for the models must be opaque, but may have any color.
+
+In order to compete in all main challenges, a minimum number of 3 mockup
+machines are required. On higher difficulty some challenges may require
+4 mockup stations and some secondary challenges require 7 stations.
+
+\subsection{Remote Setup}
+In case a competition is carried out remotely, a proper local setup has to
+be established and approved by the \ac{OC}.
+Requirements include a proper camera setup that covers the field sufficiently,
+such that external viewers can verify the integrity of each challenges,
+as well as an approval for every mockup machine and robot that is used.
+After registration the \ac{OC} will verify the field setup in a video call.
+The validation call will be scheduled individually for each team
+to account for timezones.
+
+\section{Game Play}
+\subsection{Competition Scope}
+Each team can attempt a challenge by specifying a 30 minute time slot along
+with the challenge in advance of the respective competition day.
+The announcement procedure is managed by the \ac{OC}.
+Each slot can be used to do at most one challenge and is split in two phases:
+1. In the \texttt{SETUP} phase the participants ensure that
+\begin{itemize}
+	\item the video stream is setup via Zoom,
+	\item the RefBox is started with the appropriate settings for the challenge,
+	\item the machines and robots are placed on the field according to the rules
+	\item and the robots are started.
+\end{itemize}
+The setup must be completed within 10 minutes, otherwise the attempt is counted
+as a failure.
+Then the \texttt{PRODUCTION} phase starts by instructing the RefBox accordingly.
+During the production phase the team has 20 minutes to solve the challenge.
+
+A team may retry the challenge by setting the RefBox back to the setup phase
+and then starting the production phase again.
```

Comment:
What happens to the machines, robots, will everything be reset? This should be stated explicitly.